hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirRenameOp.java

```diff
@@ -249,7 +249,7 @@ static RenameResult renameToInt(
     String dst = dstArg;
     if (NameNode.stateChangeLog.isDebugEnabled()) {
       NameNode.stateChangeLog.debug("DIR* NameSystem.renameTo: with options -" +
-          " " + src + " to " + dst);
+          " {} to {}, options={}", src, dst, Arrays.toString(options));
```

Comment:
How about changing the code as bellow?
```
NameNode.stateChangeLog.debug("DIR* NameSystem.renameTo: with options {} - {} to {}",
        Arrays.toString(options), src, dst);
```

Reply:
I agree.
Here are some audit logs:
`
2022-10-14 11:16:30,825 [Listener at localhost/51113] INFO FSNamesystem.audit (FSNamesystem.java:logAuditMessage(8853)) - allowed=true ugi=hdfs (auth:SIMPLE) ip=null cmd=rename ( options=[NONE]) src=/testNamenodeRetryCache/testRename2/src dst=/testNamenodeRetryCache/testRename2/target perm=hdfs:supergroup:rwxrwxrwx proto=null
`
Should we keep a consistent norm?
New format:
`
2022-10-14 11:20:18,813 [Listener at localhost/58086] DEBUG hdfs.StateChange(FSDirRenameOp.java:renameToInt(256)) - DIR* NameSystem.renameTo: with options=[NONE] /testNamenodeRetryCache/testRename2/src to /testNamenodeRetryCache /testRename2/target
`

Reply:
ok

Reply:
Can remove `if (NameNode.stateChangeLog.isDebugEnabled()) {`

Reply:
Done, waiting for UT.

Reply:
`
if (NameNode.stateChangeLog.isDebugEnabled()) {
     ......
}
`
In Hadoop, this phenomenon seems to be a lot.